{
  "comments": [
    {
      "key": {
        "uuid": "1aa4ed85_3c4bf788",
        "filename": "yardstick/common/constants.py",
        "patchSetId": 4
      },
      "lineNbr": 27,
      "author": {
        "id": 1402
      },
      "writtenOn": "2017-07-17T04:28:43Z",
      "side": 1,
      "message": "please use caps,  CONF for module-level scope.\n\n\nplease use  parse_yaml\n\nfrom yardstick.common.utils import parse_yaml",
      "revId": "610529f210e1f6379822562441bc69c9a6aaba03",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "1aa4ed85_1c4abb80",
        "filename": "yardstick/common/constants.py",
        "patchSetId": 4
      },
      "lineNbr": 27,
      "author": {
        "id": 5437
      },
      "writtenOn": "2017-07-17T04:33:33Z",
      "side": 1,
      "message": "I try to import it from utils, but get a loop import error, and I can\u0027t import constants. But it seems in your patch is ok. I have no idea...",
      "parentUuid": "1aa4ed85_3c4bf788",
      "revId": "610529f210e1f6379822562441bc69c9a6aaba03",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "1aa4ed85_fc447f75",
        "filename": "yardstick/common/utils.py",
        "patchSetId": 4
      },
      "lineNbr": 276,
      "author": {
        "id": 1402
      },
      "writtenOn": "2017-07-17T04:33:47Z",
      "side": 1,
      "message": "ah, so this is creating a circular import dependency.\n\nThen this should be moved to a different file.  It is not a general utility function, it is specific to API server.",
      "revId": "610529f210e1f6379822562441bc69c9a6aaba03",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "1aa4ed85_dc5bc352",
        "filename": "yardstick/common/utils.py",
        "patchSetId": 4
      },
      "lineNbr": 277,
      "author": {
        "id": 1402
      },
      "writtenOn": "2017-07-17T04:33:47Z",
      "side": 1,
      "message": "close() needs to be in a try/finally block, or in a closing() context manager\n\nhttps://docs.python.org/3/library/contextlib.html#contextlib.closing\n\n with closing(socket.socket(...)) as s:\n    while True:\n    if s.connect_ex(...)",
      "revId": "610529f210e1f6379822562441bc69c9a6aaba03",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}